deploy/infrastructure/modules/terraform-aws-dss/terraform.dev.example.tfvars

```diff
@@ -0,0 +1,25 @@
+# This file is an example, please adapt it to your configuration.
+# See TFVARS.md for full set of variables and related descriptions.
+
+# AWS account
+aws_region = "eu-west-1"
+
+# DNS
+aws_route53_zone_id  = "Z01551234567890123456"
```

Comment:
Only `aws_route53_zone_id` has to do with DNS, right?  If so, let's add a line break after it to clarify that `app_hostname` and `crdb_hostname_suffix` are needed regardless of whether DNS management is being performed by terraform.

Reply:
For me the hostnames were part of the DNS configuration in general. I added sections DNS management and  hostnames to separate the two sets of variables.